src/pages/Users/Users.module.scss

```diff
@@ -1,33 +1,31 @@
-@import "../../app/variables.scss";
+@import '../../app/variables.scss';
 
 .usersPage {
-  width: $contentContainerWidth;
-  margin-top: 60px;
-  display: flex;
-  flex-direction: column;
-  align-items: start;
-  justify-content: start;
-  height: 100vh;
-  background-color: #fff;
+	margin-top: 60px;
+	display: flex;
+	flex-direction: column;
+	align-items: start;
+	justify-content: start;
+	background-color: #fff;
 
-  &Options {
-    display: flex;
-    padding-left: 20px;
-    margin-bottom: 30px;
+	&Options {
```

Comment:
really not sure about the amount of space from the beginning of the lines...

Reply:
um, hope last var vill be better )

Reply:
Well, this styles about this part:
![image](https://github.com/user-attachments/assets/2d198ae2-3319-49a1-bc63-b3f400c83b36)

I`m not understand wy here (on git) its marked red? but i have its on my side.